packages/monaco-editor-wrapper/src/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { WebSocketConfigOptions, WebSocketConfigOptionsUrl } from './wrapper.js';
+
+export const createUrl = (config: WebSocketConfigOptions | WebSocketConfigOptionsUrl) => {
+    let buildUrl = '';
+    if ((config as WebSocketConfigOptionsUrl).url) {
+        const options = config as WebSocketConfigOptionsUrl;
+        if (!options.url.startsWith('ws://') && !options.url.startsWith('wss://')) {
+            throw new Error(`This is not a proper websocket url: ${options.url}`);
+        }
+        options.url = options.url.replace(':80', '');
```

Comment:
If we have something like `:800` or `:801`, then this will not work as expected. Do we need to really trim of the port when explicitly passed?

Reply:
This might help: `options.url.replace(/:80(?=[^\d]|$)/, '');`. Which should only trim `:80` when it's the last thing in your url, or when there's something that follows but isn't a digit (and don't replace that character either).